flutter/build_android.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
-$ANDROID_NDK/toolchains/aarch64-linux-android-4.9/prebuilt/darwin-x86_64/bin/aarch64-linux-android-strip android/app/src/main/jniLibs/arm64-v8a/*
-flutter build apk --target-platform android-arm64,android-arm --release --obfuscate --split-debug-info ./split-debug-info
-flutter build apk ---split-per-abi --target-platform android-arm64,android-arm --release --obfuscate --split-debug-info ./split-debug-info
-flutter build appbundle --target-platform  android-arm64,android-arm --release --obfuscate --split-debug-info ./split-debug-info
+
+MODE=${MODE:=debug}
```

Comment:
default should be release, because this sh is meant to be for release originally, please do not change orignal behavior when adding new behavior.

Reply:
Ok